doc/develop/api/api_conventions.rst

```diff
@@ -0,0 +1,58 @@
+.. _api_conventions.rst:
+
+API Conventions
+###############
+
+Zephyr provides a few programming interfaces that users can use for creating their applications.
+Zephyr's API space includes the following areas:
+- C programming interface (function prototypes, structures, and macros), provided in C header files
+- Configuration system (Kconfig)
+- Device driver model (Devicetree)
+
+Other areas, such as the build system (CMake), may also be seen as Zephyr's API. However, any area
+that is not explicitly listed herein is not considered Zephyr's API as of today.
+
+API classification
+******************
+The files that define the Zephyr APIs may contain symbols intended for different usage. The intended
+API usage is designated by the API class. Zephyr APIs are classified as:
+
+- private - these APIs are intended for use within boundary of a software component such as
+            driver or subsystem. Private APIs defined in Zephyr tree are not subject to
+            :ref:`api_lifecycle`. Therefore, can be changed or removed at any time.
```

Comment:
The indenting is wrong, it should be either the following to render as a "description list"

```suggestion
- Private
          These APIs are intended for use within boundary of a software component such as
          driver or subsystem. Private APIs defined in Zephyr tree are not subject to
          :ref:`api_lifecycle`. Therefore, can be changed or removed at any time.
```

Or, to render as a regular bullet point, everything aligned after the bullet and whitespace.

```suggestion
- Private - These APIs are intended for use within boundary of a software component such as
   driver or subsystem. Private APIs defined in Zephyr tree are not subject to
   :ref:`api_lifecycle`. Therefore, can be changed or removed at any time.
```

BTW you can see the rendered doc here:  https://builds.zephyrproject.io/zephyr/pr/62198/docs/develop/api/api_conventions.html#api-conventions

Reply:
Done